components/script/dom/gpucommandencoder.rs

```diff
@@ -275,3 +361,37 @@ impl GPUCommandEncoderMethods for GPUCommandEncoder {
         )
     }
 }
+
+fn convert_buffer_cv(buffer_cv: &GPUBufferCopyView) -> wgpu_com::BufferCopyView {
+    wgpu_com::BufferCopyView {
+        buffer: buffer_cv.buffer.id().0,
+        layout: convert_texture_data_layout(&buffer_cv.parent),
+    }
+}
+
+pub fn convert_texture_cv(texture_cv: &GPUTextureCopyView) -> wgpu_com::TextureCopyView {
+    wgpu_com::TextureCopyView {
+        texture: texture_cv.texture.id().0,
+        mip_level: texture_cv.mipLevel,
+        origin: match texture_cv.origin {
+            GPUOrigin3D::RangeEnforcedUnsignedLongSequence(ref v) => wgt::Origin3d {
```

Comment:
we've got to be careful here: the user doesn't have to specify all 3 elements, it could be just `[x]`, which should mean the same as `[x, 0, 0]`, for example. Same goes for `GPUExtent3D`, but the defaults in it are `1`

Reply:
I suppose this will be the case with `GPUColor` too. But what would be the defaults in that case?

Reply:
not sure.. for color, it would make sense to default alpha to 1, but not the other components